src/CoreWCF.Primitives/src/CoreWCF/Description/ServiceCredentials.cs

```diff
@@ -31,8 +31,8 @@ public ServiceCredentials()
             clientCertificate = new X509CertificateInitiatorServiceCredential();
             serviceCertificate = new X509CertificateRecipientServiceCredential();
             windows = new WindowsServiceCredential();
-            //this.issuedToken = new IssuedTokenServiceCredential();
-            //this.secureConversation = new SecureConversationServiceCredential();
+            this.issuedToken = new IssuedTokenServiceCredential();
+            this.secureConversation = new SecureConversationServiceCredential();
```

Comment:
Remove this. This a VS code helper fix for this so you can do the entire file at once.